axelor-account/src/main/java/com/axelor/apps/account/service/invoice/InvoiceTermServiceImpl.java

```diff
@@ -405,4 +408,18 @@ public boolean checkInvoiceTermDeletionConditions(Invoice invoice) {
     }
     return false;
   }
+
+  @Override
+  @Transactional(rollbackOn = {AxelorException.class, Exception.class})
```

Comment:
This method does not throw AxelorException, and `Exception.class` already includes every exception. Here we can keep only Exception or only write `@Transactional` since this method does not throw checked exception